scripts/seed.ts

```
@@ -0,0 +1,103 @@
+  SPS15m: string;
+  SPS1h: string;
+}
+
+// Using partial to allow for fields that may or may not be present but are not required
+async function fetchPoolStats(): Promise<Partial<PoolStatsData>> {
+  console.log('Fetching pool stats...');
+  const apiUrl = process.env.API_URL || 'https://solo.ckpool.org';
+  const response = await fetch(`${apiUrl}/pool/pool.status`);
+  const data = await response.text();
+  const jsonLines = data.split('\n').filter(Boolean);
+  const parsedData = jsonLines.reduce((acc, line) => ({ ...acc, ...JSON.parse(line) }), {});
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)